.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "deps/cpp-httplib"]
+	path = deps/cpp-httplib
+	url = https://github.com/yhirose/cpp-httplib.git
+    ignore = untracked
```

Comment:
Something weird going on here formatting wise.

Reply:
Does `.gitmodules` require tab indentation?

Reply:
No, but I guess I'd like it to be consistent.